scripts/pre_commit_linter.py

```diff
@@ -202,6 +202,47 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
+    {
+        'regexp': re.compile(r'toThrow\('),
+        'message': 'Please use \'toThrowError\' instead of '
+                   '\'toThrow\'',
+        'excluded_files': (
+            # The toThrow error is standardised toThrowError and currently
```

Comment:
Can you write this so that it is in proper English? I don't understand the first sentence, and I can't tell what the "you" in the second-last line refers to.

Reply:
updated the comment in the excluded file